easybuild/easyconfigs/p/Python/Python-2.7.10-GCC-4.9.3-2.25-bare.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Python'
+version = '2.7.10'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
```

Comment:
It's more about only having things in place once, i.e. with GCCcore, and make it compatible with both `foss` and `intel`.
Right, @ocaisa?

I do wonder how much of a concern it is not having `binutils` in there when building with `GCCcore`... Not for performance, but to make sure it actually works (i.e. not relying on system binutils)